app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -2142,24 +2143,47 @@ class BrowserTabViewModel @Inject constructor(
     }
 
     private fun initializeViewStates() {
+        initializeDefaultViewStates()
+        viewModelScope.launch {
```

Comment:
why not launching the coroutine inside the `initializeViewStatesFromPersistedData()` function?

Reply:
No strong reason either way. One weak reason being that it makes it more obvious when looking at `initializeViewStates()` that the second operation is async.